scripts/target.rb

```diff
@@ -8,7 +8,9 @@ class Target
       'openstack' => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2'),
       'hyperv'    => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
       'azure'     => ImagefactoryMetadata.new('hyperv', nil, 'vhd'),
-      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box')
+      'vagrant'   => ImagefactoryMetadata.new('vsphere', 'vagrant-virtualbox', 'box'),
+      'libvirt'   => ImagefactoryMetadata.new('openstack-kvm', nil, 'qc2'),
```

Comment:
Isn't this the same as the openstack image?  Would it be simpler to just rename the openstack image or copy it instead of a full build?